frontend/src/EditTask.svelte

```diff
@@ -313,6 +321,28 @@ async function deleteTask(proceed) {
                         semesterBeginDate={$semester.begin}
                         onFromDuplicateClick={setAssignedDateToVisible}
                         onToDuplicateClick={setDeadlineToAssigned} />
+                      {#if clazz.deadline}
+                        <div class="col-md">
+                          <div class="input-group">
+                            <input
```

Comment:
Would it be possible to change the checkbox button to a more [classic](https://getbootstrap.com/docs/5.0/forms/checks-radios/#checks) style? I somehow struggle to immediately see if it's checked or not.